src/Sylius/Bundle/ResourceBundle/DependencyInjection/AbstractResourceExtension.php

```diff
@@ -316,4 +318,119 @@ protected function mapTranslations(array $classes, ContainerBuilder $container)
             }
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        if (!$container->hasExtension('sylius_resource')) {
+            throw new ServiceNotFoundException('SyliusResourceBundle must be registered in kernel.');
+        }
+
+        // If the default mapping parameter has already been defined we don't need to do anything
+        if ($container->hasParameter('sylius.translation.default.mapping')) {
+            return;
+        }
+
+        // Parse sylius_translation to get the default mapping values and assign them to
+        // 'sylius.translation.default.mapping' parameter to be used un process) method.
+        $configs = $container->getExtensionConfig('sylius_resource');
+        $config  = $this->processConfiguration(new Configuration(), $configs);
+
+        $container->setParameter('sylius.translation.default.mapping', $config['default_mapping']);
+    }
```

Comment:
This means that every bundle will have this called? All extend from this class...